x-pack/plugin/core/src/test/java/org/elasticsearch/license/TestUtils.java

```diff
@@ -274,14 +274,17 @@ public static License generateSignedLicense(String type, int version, long issue
                 .version(version)
                 .expiryDate(System.currentTimeMillis() + expiryDuration.getMillis())
                 .issueDate(issue)
-                .type(licenseType)
+                .type(licenseType )
                 .issuedTo("customer")
                 .issuer("elasticsearch")
                 .maxNodes(5);
         if (version == License.VERSION_START) {
             builder.subscriptionType((type != null) ? type : randomFrom("dev", "gold", "platinum", "silver"));
             builder.feature(randomAlphaOfLength(10));
         }
+        if ("enterprise".equals(licenseType)) {
+            builder.version(License.VERSION_ENTERPRISE).maxResourceUnits(5).maxNodes(-1);
```

Comment:
nit: I think we generally prefer separate calls for builder as it is clearer to read, or at least split them in different lines. Also maxResourceUnits can be called with a randomIntegerInRange

Reply:
Splitted them in different lines. Also updated for random integer.